web_widget_plotly_chart/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "Web Widget Plotly",
+    'summary': """Allow to draw plotly charts.""",
+    'description': """Allow to draw plotly charts.""",
+    'author': "LevelPrime srl, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/web",
+    'category': 'Web',
+    'version': '12.0.1.0.0',
+    'depends': ['base'],
+    'data': [
+        'static/src/xml/assets.xml'
```

Comment:
The `assets.xml` needs to be moved outside the `static` folder, because now Odoo will send it to the client and there is no need for that. https://github.com/OCA/OCB/blob/12.0/odoo/http.py#L1345